src/mem3_httpd.erl

```diff
@@ -44,7 +46,12 @@ handle_shards_req(#httpd{method='GET',
     couch_httpd:send_json(Req, {[
         {range, Shard},
         {nodes, Dbs}
-    ]}).
+    ]});
+handle_shards_req(#httpd{path_parts=[_DbName, <<"_shards">>]}=Req, _Db) ->
+    chttpd:send_method_not_allowed(Req, "GET");
+handle_shards_req(#httpd{path_parts=[_DbName, <<"_shards">>, _DocId]}=Req, _Db) ->
```

Comment:
@kxepal I wouldn't spend any more time on that problem right now, let's ship 2.0.